package.json

```diff
@@ -4,6 +4,8 @@
   "dependencies": {
     "@edx/cookie-policy-banner": "1.1.10",
     "@edx/edx-bootstrap": "0.4.3",
+    "@edx/edx-proctoring": "git+https://git@github.com/edx/edx-proctoring.git",
+    "@edx/mockprock": "git+https://git@github.com/edx/mockprock.git",
```

Comment:
The default config doesn't need to include mockprock. Only developers would need that. Is there a way to indicate development dependencies?

Reply:
entered https://openedx.atlassian.net/browse/EDUCATOR-3792 for this; it'll be trivially easy to do after we integrate <PROCTORING PROVIDER NAME REDACTED>